packages/application-shell/src/components/quick-access/index.js

```diff
@@ -47,7 +47,9 @@ export default class QuickAccessTrigger extends React.Component {
       // react-modal uses it for example. We want to treat those elements
       // similar to document.body.
       // See https://stackoverflow.com/a/32912224
-      event.target.getAttribute('tabindex') !== '-1'
+      event.target.getAttribute('tabindex') !== '-1' &&
+      // Do not prevent Quick Access from opening when a link has focus
+      event.target.nodeName !== 'A'
```

Comment:
Can you point to the documentation about this info?

Reply:
What do you mean? Which Info?

Reply:
`nodeName !== 'A'`

Reply:
This is the regular DOM. You can do `document.querySelector('div').nodeName` and it will return `DIV`. Are you talking about a link to MDN? I wouldn't add that as it's the DOM itself.

The documentation is at https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeName

Reply:
Ah ok, I thought it was something else. Nvm then

Reply:
No worries, I had a suspicion it's just a misunderstanding :)